src/LightsOff.cpp

```diff
@@ -182,7 +183,7 @@ struct LightsOffWidget : ModuleWidget {
 		};
 
 		menu->addChild(new MenuSeparator());
-		menu->addChild(construct<MenuLabel>(&MenuLabel::text, "Hotkey Ctrl/Cmd+Alt+X"));
+		menu->addChild(construct<MenuLabel>(&MenuLabel::text, string::f("Hotkey %s+Alt+X", RACK_MOD_CTRL_NAME)));
```

Comment:
Let me be clear.  This module is your code and I don't want to dictate how it's written.
As far as I'm concerned you can merge your own pull requests if you want.
I made you a collaborator because this is Omri's project and not mine.

Reply:
But I'm happy to keep reviewing PRs if you want.

Reply:
Thanks for pulling this in, David. I'm aware I can do it myself and I will do so, it was just a matter of time and priority during the last month.